Frameworks/CoreText/CTRun.mm

```diff
@@ -256,11 +261,15 @@ void CTRunDraw(CTRunRef run, CGContextRef ctx, CFRange textRange) {
         range.length = [string length];
     }
 
-    int numGlyphs = curRun->_characters.size();
+    int numGlyphs = curRun->_glyphAdvances.size();
     WORD* glyphs = (WORD*)IwMalloc(sizeof(WORD) * numGlyphs);
 
+    WORD* characters = (WORD*)IwMalloc(sizeof(WORD) * numGlyphs);
```

Comment:
It would probably be better to just use std::vector characters(numGlyphs); than malloc and freeing manually style-wise 

---
In reply to: [77747600](https://github.com/Microsoft/WinObjC/pull/950#discussion_r77747600) [](ancestors = 77747600,77732196)